packages/mdl-snackbar/mdl-snackbar.scss

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "mdl-animation/mixins";
+@import "mdl-theme/mixins";
+@import "mdl-typography/mixins";
+@import "./variables";
+
+/* postcss-bem-linter: define snackbar */
+.mdl-snackbar {
+  display: block;
+  display: flex;
+  position: fixed;
+  bottom: 0;
+  left: 50%;
+  justify-content: space-between;
+  transform: translate(0, 80px);
+  transition: transform .25s $mdl-animation-fast-out-linear-in-timing-function;
+  background-color: $snackbar-background-color;
+  cursor: default;
+  z-index: 3;
```

Comment:
In V1 we needed it to overlap some page contents but still be under other pieces.

Ideally, we should be able to remove this in V2 and be fine. Only add a z-index back if we hit a problem with other components conflicting.